### DISCUSSION TITLE
Audio only

### DISCUSSION THREAD

**Discussion by ahosovidaho** [mikebrady/shairport-sync#1258] [Q&A]
Is possible to set up shairport-sync for audio only? While video remains playing on iOS devices? Thank you!

**Comment by charlesomer** [comment]
Hi, I'm not quite sure I understand what you mean, this should be the normal behaviour of Shairport Sync by the sounds of it?

**Reply by ahosovidaho** [reply]
yes, this is normal behaviour, but right now I can't watch content (video) on iOS device, because I can see only text: This video is playing on "AirPlay device". 

My goal is:  watching content on iOS device while sound gets from AirPlay speaker. 

Thank you for your advice :-)

**Reply by acresp** [reply]
In the same kind of behaviour, totally comestic and not important but never understood why:
- The shairport sync devices have an other "icon" when streaming from OSX: if you click to select an other output on the Mac, you see AirPlay devices, a Pioneer receiver with AirPlay 1 have a "speaker" icon, but shairport-sync always had (AP1 or AP2) a different icon. AppleTV have also an icon representing the ATV).

<img width="294" alt="Screen Shot 2021-08-02 at 17 53 24" src="https://user-images.githubusercontent.com/57019575/127889547-ae870cb4-8727-438f-b503-4ca5862c2526.png">

**Comment by mikebrady** [comment]
Yea, I've never understood why either, but it doesn't seem to be a bug in Shairport Sync, which advertises itself always as audio-only.
